### ISSUE TITLE
Plugin throws NoClassDefFoundError (com/intellij/mock/MockVirtualFile) in Goland

### ISSUE THREAD

**Issue by tony84727** [bazelbuild/intellij#2490]
I'm using Goland `2020.2.4`:
```
GoLand 2020.2.4
Build #GO-202.8194.19, built on November 26, 2020
Runtime version: 11.0.9+11-b944.49 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 5.4.0-70-generic
GC: ParNew, ConcurrentMarkSweep
Memory: 2014M
Cores: 4
Registry: ide.completion.variant.limit=500, suggest.all.run.configurations.from.context=true, ideFeaturesTrainer.welcomeScreen.tutorialsTree=TRUE
Non-Bundled Plugins: IdeaVIM, org.jetbrains.plugins.sass, io.protostuff.protostuff-jetbrains-plugin, org.intellij.plugins.hcl, org.jetbrains.idea.clojure, name.kropp.intellij.makefile, org.toml.lang, com.jetbrains.lang.ejs, intellij.prettierJS, PythonCore, com.google.idea.bazel.ijwb, org.rust.lang
Current Desktop: ubuntu:GNOME
```

Updated the plugin to version `2021.03.23.0.1` today. The plugin threw `NoClassDefFoundError` (Stacktrace below)
And the project tree stuck at loading state:
![image](https://user-images.githubusercontent.com/45312147/113646088-afa76a00-96ba-11eb-8670-96a400783c94.png)

Stacktrace:
<details>
Following exceptions happened soon after this one, most probably they are induced.

2021-04-06T09:14:10.445
java.lang.NoClassDefFoundError: com/intellij/mock/MockVirtualFile
	at com.google.idea.blaze.base.treeview.BlazeTreeStructureProvider.keepOriginalNode(BlazeTreeStructureProvider.java:79)
	at com.google.idea.blaze.base.treeview.BlazeTreeStructureProvider.doModify(BlazeTreeStructureProvider.java:67)
	at com.google.idea.sdkcompat.general.BaseSdkCompat$TreeStructureProviderAdapter.modify(BaseSdkCompat.java:109)
	at com.intellij.ide.util.treeView.AbstractTreeStructureBase.getChildElements(AbstractTreeStructureBase.java:45)
	at com.intellij.ui.tree.StructureTreeModel.getValidChildren(StructureTreeModel.java:384)
	at com.intellij.ui.tree.StructureTreeModel.validateChildren(StructureTreeModel.java:298)
	at com.intellij.ui.tree.StructureTreeModel.getNode(StructureTreeModel.java:292)
	at com.intellij.ui.tree.StructureTreeModel.getChildren(StructureTreeModel.java:312)
	at com.intellij.ui.tree.AsyncTreeModel$CmdGetChildren.getNode(AsyncTreeModel.java:583)
	at com.intellij.ui.tree.AsyncTreeModel$Command.get(AsyncTreeModel.java:478)
	at com.intellij.ui.tree.AsyncTreeModel$Command.get(AsyncTreeModel.java:444)
	at com.intellij.util.concurrency.Invoker$Task.run(Invoker.java:316)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1110)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.lambda$runInReadActionWithWriteActionPriority$0(ProgressIndicatorUtils.java:77)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runActionAndCancelBeforeWrite(ProgressIndicatorUtils.java:153)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.lambda$runWithWriteActionPriority$1(ProgressIndicatorUtils.java:114)
	at com.intellij.openapi.progress.ProgressManager.lambda$runProcess$0(ProgressManager.java:56)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:170)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:629)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:581)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:157)
	at com.intellij.openapi.progress.ProgressManager.runProcess(ProgressManager.java:56)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runWithWriteActionPriority(ProgressIndicatorUtils.java:111)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runInReadActionWithWriteActionPriority(ProgressIndicatorUtils.java:77)
	at com.intellij.util.concurrency.Invoker.invokeSafely(Invoker.java:205)
	at com.intellij.util.concurrency.Invoker.lambda$offerSafely$0(Invoker.java:183)
	at com.intellij.util.concurrency.Invoker$Background.lambda$offer$0(Invoker.java:541)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:215)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:26)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.execute(BoundedTaskExecutor.java:194)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:207)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:183)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.ClassNotFoundException: com.intellij.mock.MockVirtualFile PluginClassLoader[PluginDescriptor(name=Bazel, id=com.google.idea.bazel.ijwb, path=/home/jerry/.local/share/JetBrains/GoLand2020.2/ijwb, version=2021.03.23.0.1)] com.intellij.ide.plugins.cl.PluginClassLoader@4d5057d4
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:95)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	... 40 more
</details>

**Comment by AlexeyGy** [comment]
Thanks for filing the issue! We'll be removing the dependency on MockVirtualFile in an upcoming PR which should take care of this.

**Comment by tony84727** [comment]
I can confirm it's fixed by #2509 (at least for Goland). Thanks.
